src/nvim/charset.c

```diff
@@ -34,6 +34,16 @@
 
 static int chartab_initialized = FALSE;
 
+static char_u chartab[256];     /* See init_chartab() for explanation */
```

Comment:
I think this needs to be `static uint8_t`. `char_u` is being slowly replaced with something more appropriate, also the chartab contents has nothing to do with characters, it is a bitflag integer.